File: aikido.cpp
#include<iostream>
using namespace std;

int main(){
    int A,H;
    cin >> A >> H;
    if(A>0){
        if(H>=120){
            cout << "marrom";
        }
        else if (120 > H && H >= 110){
            cout << "azul";
        }
        else if (110 > H && H >= 90){
            cout << "verde";
        }
        else if (90 > H && H >= 72){
            cout << "roxa";
        }
        else{
            cout << "amarela";
        }
    }else{
        cout << "branca";
    }
}

File: aikido/Aikido.py
A,H = map(int,input().split())
if A>0:
    if H>=120:
        print("marrom")
    elif 120 > H >= 110:
        print("azul")
    elif 110 > H >= 90:
        print("verde")
    elif 90 > H >= 72:
        print("roxa")
    else:
        print("amarela")
else:
    print("branca")